### ISSUE TITLE
Time delay start

### ISSUE THREAD

**Issue by Supersprint** [bambulab/BambuStudio#2864]
It would be helpful to have a time delay start feature for the printer. This would mean a print could be started when everyone is asleep, as it's quite noisy and hard to get to sleep whilst running.

**Comment by walterwongbbl** [comment]
Hi。@Supersprint 
Do you mean delaying printing?

**Comment by Supersprint** [comment]
Yes, delay printing

Regards Jon Newell


ATTENTION: This email may contain private and confidential information intended only for the use of the original recipient named above. If you are not the intended recipient of this email you are hereby notified that any use, dissemination, distribution or reproduction of this message is prohibited and any such action can contravene the NSW Privacy and Personal Information Protection Act 1998. If you have received this message in error please notify the sender immediately by return email and then permanently delete the original email including the notification of receipt of the email to the sender.

**Comment by ben-h-a** [comment]
Also think this would be a useful feature, it can be distracting even with the printer in another room.

**Comment by jkrauska** [comment]
Agreed -- my power costs are much higher (almost double)  until after midnight.  If I could "time shift" and schedule a print to start at 1am, it would save me a lot in power bills.

**Comment by praetp** [comment]
I agree this would be a nice feature. Current workaround is to add a pause to the G-code.

**Comment by RalphNoordanus** [comment]
I love to see delayed printing, where you send the print to the printer and it starts at a certain time or after a certain period of time.
The printer is in the next room, and at night it's too loud to print, so I want to start printing when we start our morning routine.

How I do it now is in the evening, I send the print to the printer. When it's done uploading, I cancel the print. The next morning, I use the Bamby handy app to start a print from my history.

**Comment by praetp** [comment]
> I love to see delayed printing, where you send the print to the printer and it starts at a certain time or after a certain period of time. The printer is in the next room, and at night it's too loud to print, so I want to start printing when we start our morning routine.
> 
> How I do it now is in the evening, I send the print to the printer. When it's done uploading, I cancel the print. The next morning, I use the Bamby handy app to start a print from my history.

@RalphNoordanus : you can also SEND the file without printing.
